### PR TITLE
Wr/oss cli

### PR DESCRIPTION
**Pull request by WillieRuemmele**
### What does this PR do?
Open Sources the sfdx-cli repository
### What issues does this PR fix or reference?
@W-8222430@

### CODE REVIEW

**Comment by shetzel** [comment]
@WillieRuemmele - you don't want to source control `./dist` right?

**Comment by WillieRuemmele** [comment]
> @WillieRuemmele - you don't want to source control `./dist` right?

@shetzel I don't believe it is... I don't think it should be

**Review by shetzel — APPROVED** [review]
That's weird.  First time I looked I saw the dist dir and compiled files.  I don't see them anymore so lgtm.

File: src/updateReachability.test.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  }).timeout(5000);
+
+  it('should not warn about updating from a custom S3 host when not set', async () => {
+    await callHook();
+    expect(warnings).to.deep.equal([]);
+    expect(errors).to.deep.equal([]);
+  }).timeout(5000);
+
+  it('should warn about updating from a custom S3 host and ask about SFM', async () => {
+    env.setS3HostOverride('http://10.252.156.165:9000/sfdx/media/salesforce-cli');
```
Comment (amphro) [inline src/updateReachability.test.ts:116]: IP needs to be changed. We also need to remove this message about SFM.

**Review by amphro — CHANGES_REQUESTED** [review]

**Review by amphro — APPROVED** [review]

**Pull request merged**